### ISSUE TITLE
Unable to open

### ISSUE THREAD

**Issue by biggreek19** [SickGear/SickGear#1374]
#### Branch version commit hash: I do not see this listed anywhere, I'm sorry. 
Python Version: 2.7.17 (v2.7.17:c2f86d86e6, Oct 19 2019, 20:49:36) [MSC v.1500 32 bit (Intel)]
Windows 10 Version 21H2

Expectation | Result

Sickgear to start on boot and work as designed 
|
Sickgear would not start on boot. Able to open sickgear directly from location without loading config file.

**Steps to reproduce:**
*1)* Attempt to start programing using command 'C:\Python27\python.exe C:\SickGear\SickGear\sickgear.py --datadir "C:\SickGear\data" '
*2)* During startup I see this error:

C:\Windows\System32>C:\Python27\python.exe C:\SickGear\SickGear\sickgear.py --datadir "C:\SickGear\data"
Starting up SickGear from C:\SickGear\data\config.ini
Traceback (most recent call last):
  File "C:\SickGear\SickGear\sickgear.py", line 738, in <module>
    SickGear().start()
  File "C:\SickGear\SickGear\sickgear.py", line 332, in start
    sickbeard.CFG = ConfigObj(sickbeard.CONFIG_FILE)
  File "C:\SickGear\SickGear\lib\configobj\__init__.py", line 1205, in __init__
    self._load(infile, configspec)
  File "C:\SickGear\SickGear\lib\configobj\__init__.py", line 1302, in _load
    raise error
ParseError: Invalid line ('\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00
Excluded: (Many more 'x00' lines)
...

Additional notes:
As this is happening on boot, I don't see logs being relevant.... Maybe my config file somehow got corrupted? This has been working for the past year + without any issues. 

Please let me know if there is anything else I can provide.

**Comment by JackDandy** [comment]
Something corrupted in your system, maybe drive failure?

You can try to reinstall SickGear.

There is not an actual issue here, therefore, as per our normal handling, this thread will now be marked invalid. Feel free to ask anything else below, even though it is closed that's no problem.

Good luck.

**Comment by JackDandy** [comment]
Something corrupted in your system, maybe drive failure?

You can try to reinstall SickGear. 

`COMMIT: UNKNOWN`

This means you did not install SG fully.

Also, it is not recommended to install apps to the root of c: drive due to system default perms issues. 

There is not an actual issue here, therefore, as per our normal handling, this thread will now be marked invalid. Feel free to ask anything else below, even though it is closed that's no problem.

Good luck.

**Comment by biggreek19** [comment]
Hello JackDandy,

Thank you for your reply. The config file is there, but I'm assuming it got corrupted. I just don't understand what happened. This is the top of the config file:

[General]
config_version = 21
branch = ""
git_remote = origin
cur_commit_hash = ""
cur_commit_branch = ""
encryption_version = 0
log_dir = Logs
file_logging_preset = DEBUG
file_logging_db = 0
socket_timeout = 30
web_host = 0.0.0.0
web_port = 8081

This has an assigned host IP, so clearly something got broken in the config file.... I wish I understood what happened. 

"This means you did not install SG fully."

I have been using SickGear for over a year without any issues. This was not a fresh install, I rebooted my computer and then SG wouldn't auto launch, so I tried to launch it myself using the command from your Setup Guide and that is when I saw the error. I was honestly more hopeful there was a way to fix the corrupted config file rather than having to rebuild my entire SG library, but I am assuming there is not. 

Thank you again for your time.


*Attached the last log from Jan 11th, maybe you can see something I missed...

[sickgear.log](https://github.com/SickGear/SickGear/files/7850344/sickgear.log)

**Comment by biggreek19** [comment]
Hello again,

I was able to fix this issue.... Going to post here in case someone finds this in a search down the road....

When trying to run command:
C:\Python27\python.exe C:\SickGear\SickGear\sickgear.py --datadir "C:\SickGear\data" 

I got an error that the config file was essentially corrupted. 

I found if I go to this location:  SickGear/Data/Config.ini I get the null/null/null 
Which is what I was seeing when it was posting the \x00 messaged

However, if I open the config.ini.v20 I see a correct config file with a commit_hash listed. 

'
[General]
config_version = None
branch = master
git_remote = origin
cur_commit_hash = 886585816f7b1177ff7d18375a93fb40f7ab0a5d
cur_commit_branch = ""
encryption_version = 0
log_dir = Logs
file_logging_preset = DEBUG
file_logging_db = 0
'

I made a backup of the .v20 version, removed the config.ini file and replaced it with the .20 and just re-named it as config.ini. Now SG was able to run successfully and it is booting with the OS as expected. 

Not sure what happened or why the config file got corrupted, but that seems to have been the issue. Glad I was able to find a backup and not have to re-build everything from scratch.
